improve tracibility of error message

nhorvath opened the pull request:
The error message "Ambiguous container document reference" doesn't provide a source generating it. When I encountered it it just said "Ambiguous container document reference: labels" and labels is very common in my app so I had no idea where it was originating. My issue was solved by adding @module to all the doc blocks in my labels service (usually just adding to the overall one is sufficient).

Anyway, this would help track down where the error is coming from.

nhorvath commented:
I signed it!

petebacondarwin commented:
@nhorvath - thanks for this PR. I am surprised that this adds anything that is not already there, since the `createDocMessage` service should already put the `doc.id` into the message. See https://github.com/angular/dgeni-packages/blob/master/base/services/createDocMessage.js#L6

petebacondarwin commented on ngdoc/processors/memberDocs.js:
```diff
@@ -34,15 +34,15 @@ module.exports = function memberDocsProcessor(log, getDocFromAlias, createDocMes
             containerDocs = getDocFromAlias(_.template('${module}.${memberof}')(doc), doc);
             if ( containerDocs.length !== 1 ) {
-              log.warn(createDocMessage('Ambiguous container document reference: '+ doc.memberof));
+              log.warn(createDocMessage(doc.id +' - Ambiguous container document reference: '+ doc.memberof));
```
Ahah! I see the problem.  This call to `createDocMessage` does not pass the `doc` parameter.

petebacondarwin replied:
Can you change your PR to add this parameter instead of prefixing the message with `doc.id`?

nhorvath replied:
Sure.

nhorvath commented:
@petebacondarwin I reverted my change and added the doc object. The message is much more helpful now.

> info:    running processor: memberDocsProcessor
warn:    Ambiguous container document reference: symbol - doc "symbol#method:lookup" (method)  - from file "src/services/symbol.js" - starting at line 110, ending at line 120

petebacondarwin commented:
Great! I will squash and fix up the commit messages (in accordance with our [requirements](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit)) when I merge unless you are able to do that first.

nhorvath commented:
Sorry I don't know how to edit commit messages

On Wed, Jul 12, 2017 at 9:13 AM, Pete Bacon Darwin <notifications@github.com
> wrote:

> *@petebacondarwin* approved this pull request.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/angular/dgeni-packages/pull/235#pullrequestreview-49483806>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAKE3NqGJqw7NRiKKeCB9qKctpIy9Y_lks5sNMZqgaJpZM4OTddp>
> .
>
